src/components/select-user/index.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import truncate from 'lodash/truncate';
+import { CloseIcon } from '../icons';
+
+
+const Wrapper = styled.div`
+    display: flex;
+    align-items: center;
+    min-width: 350px;
+    margin-bottom: 16px;
+`;
+
+const UserImage = styled.div`
+    flex: 0;
+    text-align: left;
+`;
+
+const UserPhoto = styled.figure`
+    width: 32px;
+    height: 32px;
+    border-radius: 100%;
+    overflow: hidden;
+    margin: 0;
+
+    img {
+        max-width: 100%;
+    }
+`;
+
+const UserName = styled.div`
+    flex: 1 1 30%;
+    padding-top: 7px;
+    padding-left: 16px;
+    padding-bottom: 6px;
+    text-align: left;
+    font-family: Roboto;
+    font-size: 16px;
+    font-weight: normal;
+    font-style: normal;
+    font-stretch: normal;
+    line-height: normal;
```

Comment:
Values like `normal`, we normally don't use, they will be the default of the user.